langchain4j-core/src/main/java/dev/langchain4j/model/chat/observability/ChatLanguageModelListener.java

```diff
@@ -0,0 +1,48 @@
+package dev.langchain4j.model.chat.observability;
+
+import dev.langchain4j.Experimental;
+
+/**
+ * TODO
+ */
+// TODO name: ChatLanguageModelObserver?
+// TODO package
+@Experimental
+public interface ChatLanguageModelListener {
```

Comment:
We will need to call the listener from different frameworks, like OTel and Micrometer. Iterating through the listeners would be nice.

Reply:
Totally makes sense, will add!